src/by/teachmeskills/homework/hw_10032023/Car.java

```diff
@@ -0,0 +1,50 @@
+package by.teachmeskills.homework.hw_10032023;
+
+import by.teachmeskills.homework.hw_10032023.exceptions.CarException;
+
+public class Car {
+    protected int price;
+    protected int speed;
+    protected String model;
+
+    public Car(int price, int speed, String model) {
+        this.price = price;
+        this.speed = speed;
+        this.model = model;
+    }
+
+    public Car() {
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
+    }
+
+    public int getSpeed() {
+        return speed;
+    }
+
+    public void setSpeed(int speed) {
+        this.speed = speed;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public void start() throws CarException {
+        int num = (int) (Math.random() * 20);
+        if (num % 2 == 0) {
+            throw new CarException("try again");
```

Comment:
Сообщение должно быть более информативным.
throw new CarException("Car is not started because random number is even. Please, try again.");
или просто throw new CarException("Car is not started because random number is even");